### PR TITLE
#159700744 Add center_id to asset schema

### PR DESCRIPTION
**Pull request by Victoradukwu**

### CODE REVIEW

**Comment by GbengaOshinaga** [comment]
@Victoradukwu Also update the asset seed data to include the `center_id`

File: api/utilities/validators/asset_validator.py [hunk 1]
```diff
@@ -77,6 +77,20 @@ def asset_data_validators(request, edit):
...
+            {
+                "message": serialization_errors["invalid_center_id"]
+            }, 400)
+
+    if ('centerId' in request_data
+            and not Center.get(request_data['centerId'])):
+        raise ValidationError(
+            {
+                "message": serialization_errors["center_not_found"]
+            }, 400)
```
Comment (appcypher) [inline api/utilities/validators/asset_validator.py]: I believe the status code here should be `404`

**Comment by SEUNAGBEYE** [comment]
@Victoradukwu Please remove the `space` between `# 159700744`

**Comment by Victoradukwu** [comment]
@SEUNAGBEYE @appcypher @GbengaOshinaga All requested changes have been fixed.

**Pull request merged**